keyboards/ergodox/config.h

```diff
@@ -11,8 +11,8 @@
 
 /* define if matrix has ghost */
 //#define MATRIX_HAS_GHOST
-
-#define TAPPING_TERM    200
+#define PERMISSIVE_HOLD
+#define TAPPING_TERM    125
```

Comment:
Is there a reason why you're changing the default config? This change would affect _all_ Ergodox users. I suggest moving this to the `config.h` in your personal keymap.